libp2p/muxers/mplex/lpchannel.nim

```diff
@@ -106,19 +117,19 @@ proc closeMessage(s: LPChannel) {.async.} =
   withEOFExceptions:
     withWriteLock(s.writeLock):
       trace "sending close message", id = s.id,
```

Comment:
if you place the variables at the next line indented by 2 spaces, you don't have to change every time the string changes

Reply:
not sure what you mean :)

Reply:
```
trace "..."
  id = ...
```
less reformatting over time